IbisLib/scenemanager.cpp

```diff
@@ -623,6 +623,12 @@ void SceneManager::AddObjectUsingID( SceneObject * object, SceneObject * attachT
         this->SetupInAllViews( object );
         object->PreDisplaySetup();
 
```

Comment:
We should also take into account all non-system objects. For example, if the first object to be added is a PolyDataObject, this should also reset the view.